ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -780,7 +784,20 @@ def array_indices_axis(
     indices = indices[0]
     if disable_random_axis:
         return [x_dtype, indices_dtype], x, indices
-    return [x_dtype, indices_dtype], x, indices, axis, batch_dims
+
+    if values is not None:
```

Comment:
This function is used by other tests. In order not to break them you should maintain the original output on `values = None`:
 ```
if values is None:
    return [x_dtype, indices_dtype], x, indices, axis, batch_dims
else:
      ....
```

Reply:
Okay sure! Thanks.